docs/modules/plugins/pages/plugin-aws-lambda.adoc

```diff
@@ -0,0 +1,54 @@
+= AWS Lambda Plugin
+
+The plugin provides functionality to interact with https://aws.amazon.com/lambda/[AWS Lambda].
+
+== Installation
+
+.build.gradle
+[source,gradle,subs="attributes+"]
+----
+implementation(group: 'org.vividus', name: 'vividus-plugin-aws-lambda', version: '{current-version}')
+----
+
+include::partial$aws.adoc[]
+
+== Steps
+
+=== Invoke a Lambda function
+
+Invoke a Lambda function by providing the function name and the payload to pass to the Lambda function.
+
+[source,gherkin]
+----
+When I invoke Lambda function `$functionName` with payload `$payload` and save result to $scopes variable `$variableName`
```

Comment:
Maybe `Lambda` -> `AWS λ`

Reply:
`λ` will be painful for users

Reply:
I would even say, not everyone knows what is the name of this letter

Reply:
So, maybe AWS Lambda then ?

Reply:
for S3 we use just S3, also there is no any Lambda which is not AWS

Reply:
BTW, the same for: Kafka vs Apache Kafka

Reply:
https://www.lambdatest.com/